CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required (VERSION 3.12)
+project(az)
+add_subdirectory(sdk/storage/storage)
```

Comment:
Why is /storage repeated, or is this just temporary to give the build system something to build and will be removed later?

Reply:
I'm following a pattern from other SDKs. AFAIK, some services may have several libraries/projects etc.